package.json

```diff
@@ -104,7 +104,7 @@
   "eslintConfig": {
     "root": true,
     "parserOptions": {
-      "ecmaVersion": 5
+      "ecmaVersion": 6
```

Comment:
and unfortunately I can't change versions of the parser per file, it's a project level thing.